pkg/operator/ceph/cluster/monitoring.go

```diff
@@ -87,7 +87,7 @@ func (c *ClusterController) configureCephMonitoring(cluster *cluster, clusterInf
 	// note: the error return below is ignored and is expected to be removed from the
 	//   bucket library's `NewProvisioner` function
 	bucketController, _ := bucket.NewBucketController(c.context.KubeConfig, bucketProvisioner)
-	go bucketController.Run(cluster.stopCh)
+	go bucketController.Run(cluster.stopCh) //nolint, we are not expecting any error here
```

Comment:
Why not?

Reply:
we calling the built-in method `Run` and also having goroutine. Better to use `//nolint` IMO.

Reply:
No, but why aren't we expecting any error here?

Reply:
I was thinking of suppressing, Now, checking for error. Thanks!